code/datums/brain_damage/creepy_trauma.dm

```diff
@@ -0,0 +1,117 @@
+/datum/brain_trauma/special/creep
+	name = "Erotomania"
+	desc = "Patient has a subtype of delusional disorder, becoming irrationally attached to someone."
+	scan_desc = "severe erotomaniac delusions"
+	gain_text = ""
+	lose_text = "<span class='warning'>You no longer feel so attached.</span>"
+	can_gain = TRUE
+	resilience = TRAUMA_RESILIENCE_SURGERY
+	var/mob/living/obsession
+	var/datum/objective/spendtime/attachedcreepobj
+	var/datum/antagonist/creep/antagonist
+	var/viewing = FALSE //it's a lot better to store if the owner is watching the obsession than checking it twice between two procs
+
+	var/total_time_creeping = 0 //just for roundend fun
+	var/time_spent_away = 0
+	var/obsession_hug_count = 0
+
+/datum/brain_trauma/special/creep/on_gain()
+
+	//setup, linking, etc//
+	..()
+	if(!obsession)//admins didn't set one
+		obsession = find_obsession()
+		if(!obsession)//we didn't find one
+			lose_text = ""
+			qdel(src)
+	owner.apply_status_effect(STATUS_EFFECT_INLOVE, obsession)
```

Comment:
```suggestion
	gain_text = "<span class='warning'>You feel a strange attachment to [obsession].</span>"
	owner.apply_status_effect(STATUS_EFFECT_INLOVE, obsession)
```

Reply:
I thought this would be annoying to do but it's really moving the ..() further down, sick

Reply:
i did this, it actually looks pretty great